parm/config/config.landanl

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -x
```

Comment:
Not sure we are doing an `-x` in the config scripts. 
```suggestion
#!/usr/bin/env bash
```

Reply:
Made changes as suggested.